src/lib/markets/tx.ts

```diff
@@ -268,6 +132,25 @@ export async function buildModifyPositionTx({
   }
 }
 
+export type BuildModifyPositionTxArgs = {
```

Comment:
Should we move this to the markets module instead since it's not used in this file?